utils/src/utils/AzExtFsExtra.ts

```diff
@@ -73,6 +73,12 @@ export namespace AzExtFsExtra {
         await workspace.fs.writeFile(uri, Buffer.from(contents));
     }
 
+    export async function appendFile(resource: Uri | string, contents: string): Promise<void> {
+        const uri = convertToUri(resource);
+        const existingContent = await AzExtFsExtra.readFile(uri);
+        await AzExtFsExtra.writeFile(uri, existingContent + '\r\n\r\n' + contents);
```

Comment:
Do we need to add the newlines though? Seems like separator should be the decision of the caller.

Reply:
We don't have to, I was emulating what `fs.appendFile` does though. We could have an option for "separator" though,  I'd like to default to `\r\n\r\n` since that is the default behavior of `fs.appendFile`.

Reply:
I think that's a good middle ground.